packages/api-key-stamper/src/index.ts

```diff
@@ -78,3 +78,7 @@ export class ApiKeyStamper {
     };
   }
 }
+
+import { pointDecode } from "./tink/elliptic_curves";
```

Comment:
I believe you can consolidate this into one line: `export { pointDecode } from "./tink/elliptic_curves";`

Reply:
Done